test/connector.ts

```diff
@@ -72,11 +73,11 @@ t.test('Connector invalid type error', async t => {
   t.rejects(
     connector.getOptions({
       ipType: 'foo' as IpAdressesTypes,
-      authType: AuthTypes.PASSWORD,
+      authType: 'PASSWORD' as AuthTypes,
```

Comment:
Let's avoid `as`. See https://google.github.io/styleguide/tsguide.html#type-and-non-nullability-assertions.

Reply:
in tests it's not so bad if needed (to use `as`) but yeah, in this specific case here though, it seems to me that it's easier to import the types and use them directly